internal/flags/flags.go

```diff
@@ -388,19 +390,45 @@ func GetSecretsFromFiles(rootCmd *cobra.Command) {
 
 // getSecretFromFile will check if the flag contains a reference to a file; if it does, replaces the value of the flag with the contents of the file.
 func getSecretFromFile(flags *pflag.FlagSet, secret string) {
-	value, err := flags.GetString(secret)
-	if err != nil {
-		log.Error(err)
-	}
-	if value != "" && isFile(value) {
-		file, err := ioutil.ReadFile(value)
-		if err != nil {
-			log.Fatal(err)
-		}
-		err = flags.Set(secret, strings.TrimSpace(string(file)))
+
+	flag := flags.Lookup(secret)
+	if flags.Lookup(secret).Value.Type() == "stringArray" {
+
+		values, err := flags.GetStringArray(secret)
 		if err != nil {
 			log.Error(err)
 		}
+
+		for _, value := range values {
+			if value != "" && isFile(value) {
+				file, err := ioutil.ReadFile(value)
+				if err != nil {
+					log.Fatal(err)
+				}
+
+				flag.Changed = false
+				rows := bytes.Split(file, []byte{'\n'})
+
+				for _, row := range rows {
```

Comment:
When running locally, I see 2 problems with this:
1) if the file ends in an empty line (something I always do, for example), it will add "" to the StringArray which causes Watchtower to crash with the error `FATA[0000] Failed to initialize Shoutrrr notifications: error initializing router services: unknown service scheme ''`
2) Even if I **don't** add an extra empty line at the end of the file, I'm still seeing that error. When debugging, I see the following (with two lines in the file):
![image](https://user-images.githubusercontent.com/8885250/84566374-c61db700-ad70-11ea-9730-a94d9c7d27ca.png)
Seems to be caused by how byte.Split() works.

Long story short: I think you'll need to add a `len(row) != 0` to make sure this works :)